travis/main.sh

```diff
@@ -40,6 +38,14 @@ check_git_status() {
 build_autotools() {
 	v --fold="bootstrap" ./bootstrap.sh
 
+	case ${TRAVIS_BUILD_TYPE} in
+	sanitizers)
+		SANITIZE_FLAGS="-fsanitize=address -fsanitize=leak -fsanitize=alignment -fsanitize=bool -fsanitize=enum -fsanitize=float-cast-overflow -fsanitize=float-divide-by-zero -fsanitize=function -fsanitize=integer-divide-by-zero -fsanitize=null -fsanitize=object-size -fsanitize=return -fsanitize=signed-integer-overflow -fsanitize=unreachable -fsanitize=unsigned-integer-overflow"
+		v --fold="set_cflags" export CFLAGS="$SANITIZE_FLAGS"
+		v --fold="set_ldflags" export LDFLAGS="$SANITIZE_FLAGS"
```

Comment:
there should be no need to set `LDFLAGS` as the build should use `CFLAGS` when linking in addition to `LDFLAGS`